packages/http/fetch/src/middlewares/browser/middlewareFactory.ts

```diff
@@ -27,10 +16,14 @@ export class MiddlewareFactory {
 	 * Returns the default middleware chain an array with the  middleware handlers
 	 * @returns an array of the middleware handlers of the default middleware chain
 	 */
-	public static getDefaultMiddlewareChain(customFetch: (request: string, init: RequestInit) => Promise<Response> = fetch as any): Middleware[] {
+	public static getDefaultMiddlewares(customFetch: (request: string, init: RequestInit) => Promise<Response> = fetch as any): Middleware[] {
 		// Browsers handles redirection automatically and do not require the redirectionHandler
-
-		const middlewareArray: Middleware[] = [new RetryHandler(), new ParametersNameDecodingHandler(), new UserAgentHandler(), new HeadersInspectionHandler(), new CustomFetchHandler(customFetch)];
-		return middlewareArray;
+		return [
+            new RetryHandler(),
```

Comment:
Just to confirm here, the reason we don't have the `RedirectHandler` in the browser config is because the browser will do that for us automatically. Yes?

Reply:
Yes, the browser should handle the redirects